app/web/features/communities/CommunityPage/JoinCommunityButton.tsx

```diff
@@ -59,17 +58,6 @@ export default function JoinCommunityButton({
         loading={isLoading}
         variant={community.member ? "outlined" : "contained"}
         onClick={() => (community.member ? leave.mutate() : join.mutate())}
-        sx={{
```

Comment:
Have the styling for this somehow been baked in/implemented somewher else?

Reply:
I copy pasta'd button style when I did the MUI upgrade and Jesse pointed out this one looked weird. When I took that style out it looks better again. I think it's just a leftover mistake from when I did the MUI upgrade.